assets/src/amp-validation/amp-validation-error-edit-screen.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+
+domReady( () => {
+	const errorDetailList = document.querySelector( 'details > dl.detailed' );
+
+	addAdditionalErrorDetails( errorDetailList );
+	unwrapErrorDetails( errorDetailList );
```

Comment:
As opposed to doing this with JS can not the `details` just be omitted when PHP generates it from the start? As it stands right now, there is a FODC (flash of `detail`'ed content)

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4382/commits/34e3e806433000e0758ff0af68dcde6dc1b94818.